### PR TITLE
Documenting the project in README.md file

### PR DESCRIPTION
**Pull request by salindholm**
## Changes proposed in this pull request:
* Modifying README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,48 @@
...
+- [Craft Academy](https://www.craftacademy.se/) coaches: Thomas Ochman, Emma-Maria Thalen
+
+## License
+MIT License
+
+## Authors
+- [Sanne Lindholm](https://github.com/salindholm)
+- [Kyungin Na](https://github.com/KyunginNa)
+- [Viktor Karlsson](https://github.com/ViktorHek)
+- [Erik](https://github.com/Ehugo2000)
```
Comment (KyunginNa) [inline README.md]: Found out what Erik's full name is! [Erik Hugoson]

File: README.md [hunk 2]
```diff
@@ -1,24 +1,48 @@
...
+
+## Description
+This is the API to our project Kitty News. Here we are authenticating users, storing articles, saving all our data and also making calls to external APIs such as Stripe. This is built with Ruby on Rails. We are using Amazon Web Services to store our images.
+- User registration
+- User authentication(Log-in/Log-off)
+- Yearly subscription plan
+- Display articles in a category
+
+## The code
+Deployed App: (https://kitty-news.herokuapp.com/) </br>
```
Comment (KyunginNa) [inline README.md]: We can delete this line as we discussed before :)

File: README.md (hunk 1 above)
Comment (salindholm) [reply README.md]: Will add that!

File: README.md (hunk 2 above)
Comment (salindholm) [reply README.md]: The heroku deployed address is not showing any where else so lets keep this!

**Pull request merged**